charts/signoz/values.yaml

```diff
@@ -422,6 +422,13 @@ queryService:
         endpoint: /ping
         waitMessage: "waiting for clickhouseDB"
         doneMessage: "clickhouse ready, starting query service now"
+      resources:
+        requests:
+          cpu: 100m
+          memory: 100Mi
+        limits:
+          cpu: 100m
+          memory: 100Mi
```

Comment:
can you comment this section?

We can support for resource configurations for init containers, but keeping it empty would be wise as users can tweak as per their need.

Moreover, init containers consumes very less resources.
Also, chart test workflow environment has limited resources and it would fail if more resources are allocated.

Reply:
Updated. Thanks for the review.